configuration/pih/scripts/global/zl.js

```diff
@@ -93,28 +93,31 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
  * and that the "end date" cannot be before the "start date" by updating the min and max dates of the respective
  * datepickers
  */
-function setUpDatepickerStartAndEndDateValidation(startDateWidgetId, endDateWidgetId){
+function setUpDatepickerStartAndEndDateValidation(){
 
-    const startDatepicker = getField(startDateWidgetId + '.value');
-    const endDatepicker = getField(endDateWidgetId + '.value');
+    jq(".startDateEndDate").each(function (j, domEl) {
 
-    if (startDatepicker) {
-      startDatepicker.change(function () {
-        let startDate = getField(startDateWidgetId + '.value').datepicker('getDate');
-        if (startDate) {
-          endDatepicker.datepicker('option', 'minDate', startDate);
+        const startDatepicker = jq(jq(this).find('.startDate'));
+        const endDatepicker = jq(jq(this).find('.endDate'));
+
+        if (startDatepicker) {
+            startDatepicker.change(function () {
+                let startDate = startDatepicker.find('input[type=text]').datepicker('getDate');
```

Comment:
there's a risk here that if the implementation of the HTML Form Entry datepicker changes, and doesn't have an underlying element input of type=text, this might break, but I agree that is a small risk.